src/app/Providers/RouteServiceProvider.php

```diff
@@ -17,28 +19,62 @@ class RouteServiceProvider extends ServiceProvider {
     /**
      * Define your route model bindings, pattern filters, etc.
      *
-     * @param  \Illuminate\Routing\Router  $router
      * @return void
      */
-    public function boot(Router $router)
+    public function boot()
     {
-        parent::boot($router);
-
         //
+
+        parent::boot();
     }
 
     /**
      * Define the routes for the application.
      *
-     * @param  \Illuminate\Routing\Router  $router
      * @return void
      */
-    public function map(Router $router)
+    public function map()
+    {
+        $this->mapApiRoutes();
+
+        $this->mapWebRoutes();
+
+        //
+    }
+
+    /**
+     * Define the "web" routes for the application.
+     *
+     * These routes all receive session state, CSRF protection, etc.
+     *
+     * @return void
+     */
+    protected function mapWebRoutes()
     {
-        $router->group(['namespace' => $this->namespace], function($router)
-        {
-            require app_path('Http/routes.php');
+        Route::group([
```

Comment:
__[phpmd]__
* Avoid using static access to class '\Illuminate\Support\Facades\Route' in method 'mapWebRoutes'.

<!-- Comment by [SideCI](https://sideci.com) -->